src/Pages/AddWork/work.tsx

```diff
@@ -1,3 +1,78 @@
-import './work.css'
+import React from 'react';
+import './work.css';
 
-export const Work: React.FC = () => <></>;
+export const Work: React.FC = () => {
+  return (
+    <div className="work-page">
```

Comment:
It would be better to make React components.

Reply:
Sure. Tomorrow 😇